src/components/blog/__tests__/OptimizedImage.test.jsx

```
@@ -189,14 +190,14 @@ describe('OptimizedImage Component', () => {
   describe('Development Logging', () => {
-    const originalEnv = import.meta.env;
+    const originalEnv = env;
 
     beforeEach(() => {
-      import.meta.env.DEV = true;
+      env.DEV = true;
     });
 
     afterEach(() => {
-      import.meta.env = originalEnv;
+      env = originalEnv;
```

Biome (2.1.2)

[error] The imported variable env is read-only

The variable is imported here

Use a local variable instead of reassigning an import.

(lint/suspicious/noImportAssign)